src/web/routes/application/address/postcode/os-places.js

```diff
@@ -10,12 +10,18 @@ const standardisePostcode = (postcode) => postcode.toUpperCase().replace(/\s/g,
 const getAddressLookupResults = async (config, postcode) => {
   const { OS_PLACES_URI, OS_PLACES_API_KEY } = config.environment
   const standardisedPostcode = standardisePostcode(postcode)
+  const requestStartDate = moment()
 
-  return request({
+  const response = await request({
     uri: `${OS_PLACES_URI}${OS_PLACES_API_PATH}?postcode=${standardisedPostcode}&key=${OS_PLACES_API_KEY}`,
     json: true,
     timeout: REQUEST_TIMEOUT
   })
+
+  const requestEndDate = moment()
+  const requestTimeInSeconds = requestEndDate.diff(requestStartDate, 'seconds', true)
```

Comment:
Trivial, but could we call these requesStartTime & EndTime, not date?

Reply:
Updated